patches/extra/ungoogled-chromium/disable-download-quarantine.patch

```diff
@@ -9,8 +9,8 @@
 -#include "components/services/quarantine/quarantine.h"
  #include "crypto/secure_hash.h"
  
- #if BUILDFLAG(IS_WIN)
-@@ -529,105 +528,6 @@ DownloadInterruptReason BaseFile::Publis
+ #if BUILDFLAG(IS_ANDROID)
```

Comment:
Is it as intended? I mean WIN → ANDROID

Reply:
Yup, that is a context line rather than a change.  
They removed that section between versions:  [99.0.4844.84](https://source.chromium.org/chromium/chromium/src/+/refs/tags/99.0.4844.84:components/download/internal/common/base_file.cc;l=28) vs [100.0.4896.60](https://source.chromium.org/chromium/chromium/src/+/refs/tags/100.0.4896.60:components/download/internal/common/base_file.cc;l=28)